app/src/main/java/io/mochahub/powermeter/exercises/ExerciseFragment.kt

```diff
@@ -57,7 +61,6 @@ class ExerciseFragment : Fragment() {
     }
 
     private fun onExerciseClick(exercise: ExerciseEntity) {
-        exerciseSharedViewModel.editExercise
```

Comment:
Not sure how I let this one sneak in here 🤣

Reply:
Good catch 👌🏾